arezoo/script.js

```diff
@@ -0,0 +1,116 @@
+//1
+let sum = 0;
+for (let i = 0; i < 20; i++) {
+  sum = sum + 1;
+  console.log(sum);
+}
+
+//2
```

Comment:
we need if statement to manage
if one "There is one bottle of beer on the wall."
more than one  "There are two bottles of beer on the wall"

Reply:
there is 1 of beer in the wall
there is 2 of beer in the wall
there is 3 of beer in the wall
there is 4 of beer in the wall
there is 5 of beer in the wall
 I wrote it with just for because of There is no such thing as checking in description number 2, it is only asked to be repeated 5 times